README.md

```diff
@@ -112,6 +112,19 @@ Specify the reporter to use.
 Possible values: `'default'`, `'json'`, and `'pmd'`.
 
 
+### options.outputPath
+
+Type: `string`
+
+Default value: `undefined`
+
+Specify the path of the output file.  
+The destination directory must already exist.
+Also make sure the file extension corresponds with the chosen [reporter](#reporter).
```

Comment:
Why that?

It doesn't matter at all, does it?
I mean I just ran the `json` reporter and wrote to `.txt` - it's not a problem at all as far as I can see?